string header
uint8 version
uint8 team
uint8 player
uint8 message
